InnoViber.Domain/Enums/UserRoles.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace InnoViber.Domain.Enums;
+
+public enum UserRoles
+{
+    Admin = 0,
```

Comment:
If we miss the set up of user role - the user will be admin by default, which is not good 
Use 'default' value for enum like None, Undefined or others

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum